test/mlir/driver/llvm.ident.mlir

```diff
@@ -2,6 +2,7 @@
 // RUN: llvm-dis %t.bc -o %t.ll
 // RUN: cat %t.ll | FileCheck %s
 // XFAIL: onnx-mlir-vendor
+// XFAIL: not-llvm-append-vc-rev
```

Comment:
This needs to be addressed a different way. Yes, it is possible to continue to add more flags and properties to disable/enable this test but it means that it is becoming pretty unpredictable if the test is going to run and whether we can rely on it to validate that everything is working correctly. The more complicated this logic gets, the less likely it will continue to work. The test should ideally run *always*.